tests/contrib/boto/test.py

```diff
@@ -0,0 +1,129 @@
+# stdlib
+import unittest
+import json
+
+# 3p
+from nose.tools import eq_
+import boto.ec2
+import boto.s3
+import boto.awslambda
+import boto.sqs
+from moto import mock_s3, mock_ec2, mock_lambda, mock_sqs
+
+# project
+from ddtrace import Pin
+from ddtrace.contrib.boto.patch import patch
+from ddtrace.ext import http
+
+# testing
+from ...test_tracer import get_dummy_tracer
+
+
+class BotoTest(unittest.TestCase):
+    """Botocore integration testsuite"""
+
+    TEST_SERVICE = "test-boto-tracing"
+
+    def setUp(self):
+        patch()
+
+    @mock_ec2
+    def test_ec2_client(self):
+        ec2 = boto.ec2.connect_to_region("us-west-2")
+        tracer = get_dummy_tracer()
+        writer = tracer.writer
+        Pin(service=self.TEST_SERVICE, tracer=tracer).onto(ec2)
+
+        # Trying describe instances command
+        ec2.get_all_instances()
+        spans = writer.pop()
+        assert spans
+        span = spans[0]
+        eq_(span.get_tag('aws.operation'), "DescribeInstances")
+        eq_(span.get_tag(http.STATUS_CODE), "200")
+        eq_(span.get_tag(http.METHOD), "POST")
+        eq_(span.get_tag('aws.endpoint'), "ec2")
+        eq_(span.get_tag('aws.region'), "us-west-2")
+
+        # Create an instance
+        ec2.run_instances(21)
+        spans = writer.pop()
+        assert spans
+        span = spans[0]
+        eq_(span.get_tag('aws.operation'), "RunInstances")
+        eq_(span.get_tag(http.STATUS_CODE), "200")
+        eq_(span.get_tag(http.METHOD), "POST")
+        eq_(span.get_tag('aws.endpoint'), "ec2")
+        eq_(span.get_tag('aws.region'), "us-west-2")
+
+        # Testing resource and service
+        eq_(span.service, "aws")
+        eq_(span.resource, "boto.command")
```

Comment:
this is not the correct resource. update this to do something similar to what botocore does. 
and also make sure the check `span.name` in this test